docker/mariadb/scripts/sql/db_schema.sql

```diff
@@ -1,20 +1,20 @@
-ALTER DATABASE CHARACTER SET utf8 COLLATE = 'utf8_unicode_ci';
```

Comment:
Should we update the existing db schemas as well?

Reply:
Yes, but only once the whole roundtrip works. The Emojis are now saved to the database which is good, we don't get errors on executing the events any more, but the data is not fed back to the client. It is only received as `?`. Ideas?

Reply:
How does the JSON object look like? Possibly wrong encoding in jdbc driver configured trough scalajdbc?